src/environments/environment.ts

```diff
@@ -4,7 +4,14 @@
 
 export const environment = {
   production: false,
-  apiUrl: 'http://localhost:34736/api/'
+  apiUrl: 'http://localhost:34736/api/',
+  httpStatusCodes: {
```

Comment:
It shouldn't be in environments because status codes should be same in dev and prod.
Consider using this lib: https://www.npmjs.com/package/http-status-codes
And create new service in app/common that provides this lib